packages/rancher-turtles/generated-changes/patch/templates/addon-provider-fleet.yaml.patch

```diff
@@ -0,0 +1,30 @@
+--- charts-original/templates/addon-provider-fleet.yaml
++++ charts/templates/addon-provider-fleet.yaml
+@@ -10,6 +10,12 @@
+     "helm.sh/hook-weight": "2"
+ spec:
+   type: addon
++  deployment:
++    containers:
++    - name: manager
++      imageUrl: "registry.rancher.com/rancher/cluster-api-addon-provider-fleet:v0.8.1"
++    - name: helm-manager
++      imageUrl: "registry.rancher.com/rancher/cluster-api-addon-provider-fleet:v0.8.1"
+   additionalManifests:
+     name: fleet-addon-config
+     namespace: '{{ .Values.rancherTurtles.namespace }}'
+@@ -31,14 +37,12 @@
+     spec:
+       config:
+         featureGates:
+-        {{- if index .Values "rancherTurtles" "rancherInstalled" }}
```

Comment:
It seems optimal to maintain the diff in the `FleetAddonConfig` in the wrapper chart, based on discussion with the team regarding https://github.com/rancher/turtles/pull/1322#discussion_r2070013837

Reply:
We can maintain the diff here, but it would be good to discuss the use-case of installing turtles via the RKE2 helm controller in future - in that scenario you can't control the ordering so it's necessary to disable the hooks which depend on Rancher, perhaps we can find a way around that (for example making the post delete hooks fail silently instead of blocking chart deletion if the chart install failed)